crates/mun_abi/src/lib.rs

```diff
@@ -25,6 +25,15 @@ pub mod prelude {
     };
 }
 
+/// Defines the current abi version
+pub const ABI_VERSION: i32 = 00_20_00;
```

Comment:
```suggestion
#[allow(clippy::zero-prefixed-literal)]
pub const ABI_VERSION: i32 = 00_02_00;
```

Reply:
I am not sure whether @baszalmstra agrees, but I think we can ignore this clippy warning. I like the general formatting of the version number into major, minor, patch.

I did change minor `20` to `02`